### DISCUSSION TITLE
Linear Regression Model

### DISCUSSION THREAD

**Discussion by vishpat** [huggingface/candle#1762] [Show and tell]
I have implemented the [Linear Regression Model](https://www.youtube.com/watch?v=UVCFaaEBnTE) using gradient descent and here is the [PR](https://github.com/huggingface/candle/pull/1756). Could someone from the candle team look it over? 

I aim to implement all of the ML algorithms taught in Andrew Ng's Stanford ML course using candle and share it with the community.

**Comment by LaurentMazare** [comment]
Thanks for putting this up. Would you mind creating a specific repo for this like candle-stanford-ml or whatever name you would feel adequate? This would seem like a better place to have this rather than in the main repo (and to help for discoverability we could add a link from the main readme, we already have an 'external resource' section for similar things).

**Reply by vishpat** [reply]
Great, I will do that. I will let you know once I have implemented all of the course algorithms. Thank you so much for candle, please keep up the great work. I am excited to see where this project goes.
